Makefile

```diff
@@ -13,8 +13,7 @@ clean:
 	npm prune
 
 lint:
-	$(JSHINT) $(SRC)
-	$(JSHINT) $(TESTS)
+	$(ESLINT) $(SRC)
```

Comment:
@seangarner for some reason when I make `eslint --fix` on the test folder, most of the tests starts to fail. I tried to understand what's going on, but without success. Don't want to burn more time on this and probably at the moment we could leave the tests as is (all are passing)